Dockerfile

```diff
@@ -1,17 +1,13 @@
-FROM quay.io/centos/centos:7
+FROM ubi9/nginx-120
```

Comment:
Please use `registry.access.redhat.com/ubi9/nginx-120`; otherwise, hub.docker.com is used.